staff/alex-barba/cv/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Àlex Barba - CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Àlex Barba <img src="myphoto.jpg" alt="àlex barba photo" height="60" width="42"></h1> 
+        <address>C/Mercat 2 Esc B 3 3, 08750 - Molins de Rei, BCN</address>
+        <p>Mobile: <a href="tel:0034618610187">+34 618 610 187</a>, email: <a href="mailto:alexander.barba@gmail.com">alexander.barba@gmail.com</a>, LinkedIn: àlex-barba </p>
+    </header>
+    <nav>
+            <a href="#we">Work Experience</a> |
+            <a href="#ed">Education</a> |
+            <a href="#la">Language</a> |
+            <a href="#sk">Skills</a>
+    </nav>
+    <section>
```

Comment:
Nice work Àlex! maybe you can add a <main> tag.
Here more information if you need: https://developer.mozilla.org/es/docs/Web/HTML/Elemento/main

Reply:
You are right. Thnx johana!